deeppavlov/core/data/utils.py

```diff
@@ -23,7 +23,6 @@
 import numpy as np
 import re
 import zipfile
-import shutil
```

Comment:
But why? `shutil.copy` is used in this file

Reply:
fixed.